osu.Game/Configuration/SessionAverageHitErrorTracker.cs

```diff
@@ -40,18 +40,18 @@ private void calculateAverageHitError(ScoreInfo? newScore)
             if (newScore.Mods.Any(m => !m.UserPlayable || m is IHasNoTimedInputs))
                 return;
 
-            if (newScore.HitEvents.Count < 10)
+            if (newScore.HitEvents.Count < 50)
```

Comment:
This requires quite a bit more gameplay to hit the point of being able to use this control. I can understand why this is the case, but time will tell as to whether users see this as an issue. Let's give it a go.